### DISCUSSION TITLE
cv2 is it essential to use DaProfiler

### DISCUSSION THREAD

**Discussion by dokoojo** [daprofiler/DaProfiler#30] [Q&A]
hello,

i can't use DaProfiler because i can't download the package cv2,

so, cv2 is it essential to run DaProfiler, or we can remove the part of the code where cv2 is needed ?

if yes, can you tell me where cv2 is used ?

thanks.

**Comment by TheRealDalunacrobate** [comment]
Hello. The cv2 package is not directly necessary to use DaProfiler, you can modify the code of this file which is in this directory modules/face_recognition
